server/src/main/java/org/kaazing/gateway/server/config/parse/translate/AddServiceNameVisitor.java

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Still suggest moving this one to EE along with other files. But if you have looked at this and think it will be useful for CE in the future I'm fine leaving it here, just comment that you have seen this